README.md

```diff
@@ -1 +1,15 @@
-If you would like to contribute simulations or experiments, please follow [these instructions](https://nmrlipids.github.io/FAIRMD_lipids/stable/dbcontribute.html).
+# FAIRMD BilayerData
+FAIRMD BilayerData is an on-github database of lipid bilayer structures operated by the [FAIRMD Lipids](https://github.com/NMRlipids/FAIRMD_lipids) Project. The database contains information about:
+- experiments, namely form factor data from Small-Angle X-ray Scattering (SAXS) experiments and order parameter data from <sup>1</sup>H-<sup>13</sup>C dipolar splitting or from <sup>2</sup>H quadrupolar splitting NMR experiments
+- MD simulations, run by different engines using various force fields
+- the molecules included in the experimental and simulation compositions, along with atom namings in different force fields and detailed metadata for each
+
+The database doesn't store raw data; it refers to public archives like Zenodo, where complete trajectories must be stored. The database structure is described in detail in the [FAIRMD Lipids documentation](https://nmrlipids.github.io/FAIRMD_lipids/stable/dbstructure.html).
```

Comment:
Line 7:

raw data -> raw simulation data

it refers -> it uses an overlay databank approach, i.e.,